multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/util/PEMToEncodedKeyConverter.java

```diff
@@ -0,0 +1,26 @@
+package org.cloudfoundry.multiapps.controller.persistence.util;
+
+import java.io.IOException;
+import java.security.KeyPair;
+import java.text.MessageFormat;
+
+import org.bouncycastle.openssl.PEMKeyPair;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter;
+import org.cloudfoundry.multiapps.controller.persistence.Messages;
+
+public class PEMToEncodedKeyConverter {
+
+    public byte[] getPrivateEncodedKey(PEMParser pemParser, JcaPEMKeyConverter converter) throws IOException {
```

Comment:
I think this method could be inside ClientKeyConfigurationHandler class or move `JcaPEMKeyConverter` initialization inside this method.